resources/dicts/thoughts/traits/medicine.json

```diff
@@ -27,42 +32,49 @@
         "Is ripping some leaves to shreds",
         "Debates becoming a warrior",
         "Gives the wrong herbs to a warrior on purpose",
-        "Wishes they weren't confined to this life",
+        "Wishes {PRONOUN/m_c/subject} weren't confined to this life",
         "Is stocking up on deathberries",
         "Is stocking up on yew berries",
         "Is stocking up on nightshade",
         "Is stocking up on watermint",
         "Put herbs in prey to give cats stomachaches",
         "Refused to treat anyone today",
-        "Is upset that they can't fight alongside the warriors",
+        "Is upset that {PRONOUN/m_c/subject} can't fight alongside the warriors",
         "Wishes to fight alongside the warriors",
-        "Thinks they'd be an amazing leader",
+        "Thinks {PRONOUN/m_c/subject}’d be an amazing leader",
```

Comment:
the apostrophe you've got here isn't ascii, if you're typing these up in google doc and then copy pasting them in, be sure to turn smart quotes off on the doc you use.  google docs likes to add silly quotations and apostrophes that will break the game text.

Reply:
tysm!! fixed this and the other mistake I had, I actually noticed the google docs thing already and was repeatedly fixing that when I first made this, seems I missed a spot !

Reply:
double checking this again rn actually as I think there may be even more I missed

Reply:
that should be all :3